presto-main/src/main/java/com/facebook/presto/sql/rewrite/DescribeInputRewrite.java

```diff
@@ -113,10 +114,10 @@ protected Node visitDescribeInput(DescribeInput node, Void context)
 
             // return the positions and types of all parameters
             Row[] rows = parameters.stream().map(parameter -> createDescribeInputRow(parameter, analysis)).toArray(Row[]::new);
-            Optional<String> limit = Optional.empty();
+            Optional<Node> limit = Optional.empty();
             if (rows.length == 0) {
                 rows = new Row[] {row(new NullLiteral(), new NullLiteral())};
-                limit = Optional.of("0");
+                limit = Optional.of(new Limit("0"));
```

Comment:
I thought you were making it 1 in the StatementAnalyzer?

Reply:
that was when the `fetch first` was provided, this is for the default limit